core-services/prow/02_config/generate-boskos.py

```diff
@@ -4,6 +4,13 @@
 
 
 CONFIG = {
+    'aws-arm64-quota-slice': {
+        # Wild guesses.  We'll see when we hit quota issues
+        'us-east-1': 10,
+        'us-east-2': 8,
+        'us-west-1': 8,
+        'us-west-2': 8,
+    },
```

Comment:
@coverprice does this seem appropriate to you?

Reply:
No idea. :) Let's wait for the quota bumps, land this, and keep an eye on it.